src/components/tricks/PostTrick.js

```diff
@@ -193,7 +202,16 @@ const PostTrick = () => {
               required
               value={difficultyLevel}
               placeholder="8"
-              onChange={(e) => setDifficultyLevel(parseInt(e.target.value))}
+              onChange={(e) => {
+                setDifficultyLevel(Math.max(0, e.target.value));
+              }}
+              onBlur={(e) => {
```

Comment:
You removed `onBlur` in the other file. Is there a reason for it still existing here?

Reply:
No, I just forgot it. Was a late night session ;)

Reply:
Removed it in ffbb5b7b11919de3c680b650d55d330920313dca